Adding documentation for private locations healthchecks (#7959)
DataDog/documentation: content/en/synthetics/private_locations/_index.md

{{< /tabs >}}

#### Set up healthchecks

> You can add a healthcheck mechanism to have your orchestrator ensure your workers are always running correctly.

Reviewer: ```suggestion
Add a healthcheck mechanism so your orchestrator can ensure the workers are running correctly.
```

> The `/tmp/liveness.date` file of private location containers gets updated after every successful poll from Datadog (every 500ms by default). The container can be considered unhealthy if no poll has been performed in a while (e.g., no fetch in the last minute).

Reviewer: ```suggestion
The `/tmp/liveness.date` file of private location containers gets updated after every successful poll from Datadog (500ms by default). The container is considered unhealthy if no poll has been performed in a while, for example: no fetch in the last minute.
```
